src/modules/client/components/customers/billing/ManualBillCreationModal.vue

```diff
@@ -64,19 +81,24 @@ export default {
     return {
       formatPrice,
       selectedBillingItem: null,
+      totalExclTaxes: 0,
     };
   },
-  computed: {
-    totalExclTaxes () {
-      return this.selectedBillingItem ? this.totalInclTaxes / (1 + this.selectedBillingItem.vat / 100) : 0;
-    },
-    totalInclTaxes () {
-      return this.selectedBillingItem ? this.newManualBill.unitInclTaxes * this.newManualBill.count : 0;
+  watch: {
+    'newManualBill.billingItemList': {
+      deep: true,
+      handler () {
+        this.totalExclTaxes = this.newManualBill.billingItemList
+          .reduce((acc, bi) => acc + this.getExclTaxes(bi.unitInclTaxes, bi.vat) * bi.count, 0);
+      },
     },
   },
   methods: {
-    nbrError (path, validations = this.$v) {
-      const val = get(validations, path);
+    getExclTaxes (inclTaxes, vat) {
+      return inclTaxes / (1 + vat / 100);
+    },
+    nbrError (path, index, validations) {
```

Comment:
on pourrait se passer de l'entrée `validations` et utiliser this.validations non ?